src/earn/EarnEntrypoint.tsx

```diff
@@ -16,8 +23,33 @@ import { Spacing } from 'src/styles/styles'
 
 export default function EarnEntrypoint() {
   const { t } = useTranslation()
-  const showUKCompliantVariant = getFeatureGate(StatsigFeatureGates.SHOW_UK_COMPLIANT_VARIANT)
 
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  const pools = useSelector(earnPositionsSelector)
+  const poolsSupplied = useMemo(
+    () => pools.filter((pool) => new BigNumber(pool.balance).gt(0)).length,
+    [pools]
+  )
+  const totalSuppliedValueUsd = useMemo(
+    () =>
+      pools.reduce(
+        (acc, pool) => {
+          const { poolBalanceInUsd } = getEarnPositionBalanceValues({ pool })
+          return acc.plus(poolBalanceInUsd)
+        },
+        new BigNumber(0) ?? null
+      ),
+    [pools]
+  )
```

Comment:
optional nit: could also do both these in a single useMemo since they have the same dependencies